modules/ROOT/pages/anypoint-mq/3.x/anypoint-mq-connector-reference.adoc

```diff
@@ -0,0 +1,526 @@
+| Output Encoding a| String |  The encoding of the operation output. |  |
+| Primary Node Only a| Boolean |  Whether this source should only be executed on the primary node when running in Cluster |  |
+| Streaming Strategy a| * <<repeatable-in-memory-stream>>
+* <<repeatable-file-store-stream>>
+* non-repeatable-stream |  Specifies to use repeatable streams. |  |
+| Redelivery Policy a| <<RedeliveryPolicy>> |  Defines a policy for processing redelivery of the message. |  |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors |  |
+|===
+
+==== Output
+[%autowidth.spread]
+|===
+|Type |Binary
+| Attributes Type a| <<AnypointMQMessageAttributes>>
+|===
+
+==== For Configurations
+* <<config>>
+
+
+
+== Types
+[[Tls]]
+=== Tls
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Enabled Protocols a| String | A comma-separated list of protocols enabled for this context. |  |
+| Enabled Cipher Suites a| String | A comma-separated list of cipher suites enabled for this context. |  |
+| Trust Store a| <<TrustStore>> |  |  |
+| Key Store a| <<KeyStore>> |  |  |
+| Revocation Check a| * <<standard-revocation-check>>
+* <<custom-ocsp-responder>>
+* <<crl-file>> |  |  |
+|===
+
+[[TrustStore]]
+=== Trust Store
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Path a| String | The location (which will be resolved relative to the current classpath and file system, if possible) of the trust store. |  |
+| Password a| String | The password used to protect the trust store. |  |
+| Type a| String | The type of store used. |  |
+| Algorithm a| String | The algorithm used by the trust store. |  |
+| Insecure a| Boolean | If true, no certificate validations will be performed, rendering connections vulnerable to attacks. Use at your own risk. |  |
+|===
+
+[[KeyStore]]
+=== Key Store
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Path a| String | The location (which will be resolved relative to the current classpath and file system, if possible) of the key store. |  |
+| Type a| String | The type of store used. |  |
+| Alias a| String | When the key store contains many private keys, this attribute indicates the alias of the key that should be used. If not defined, the first key in the file will be used by default. |  |
+| Key Password a| String | The password used to protect the private key. |  |
+| Password a| String | The password used to protect the key store. |  |
+| Algorithm a| String | The algorithm used by the key store. |  |
+|===
+
+[[standard-revocation-check]]
+=== Standard Revocation Check
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Only End Entities a| Boolean | Only verify the last element of the certificate chain. |  |
+| Prefer Crls a| Boolean | Try CRL instead of OCSP first. |  |
```

Comment:
Instructs <who?what?> to try CRL (?) before OSCP (?)

Will your audience understand these abbreviations and what "try" means?